src/multicluster/hooks/useMulticlusterProjects.ts

```diff
@@ -1,15 +1,20 @@
+import { useMemo } from 'react';
+
 import { modelToGroupVersionKind, NamespaceModel } from '@kubevirt-utils/models';
 import { getCluster } from '@multicluster/helpers/selectors';
 import useIsACMPage from '@multicluster/useIsACMPage';
 import { useFleetSearchPoll } from '@stolostron/multicluster-sdk';
 
-export type UseMulticlusterNamespacesReturnType = [
-  projectsByCluster: Record<string, K8sResourceCommon[]>,
-  loaded: boolean,
-  error: any,
-];
+export type UseMulticlusterNamespacesReturnType = {
```

Comment:
We should avoid using 'Type' and phrases like 'ReturnType' in names. We know that it's a type based on the casing and usage. The type for a hook should just be the name of the hook with the 'u' capitalized.

I know you didn't do this in this PR, but since it's being touched I'd like to fix it if you don't mind.